en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title:  "Getting started with OVH PaaS Queue"
+categories: queue
+author: guillaumebreton
+lang: en
+---
+# OVH PaaS Queue and Kafka 101
+
+Welcome to the getting started of the OVH PaaS Queue. This guide will help you to understand the core concepts behind the OVH PaaS Queue
+and how to produce your first message to the platform and consume it.
+
+## What is the OVH PaaS Queue ?
+
+The OVH PaaS Queue provides you a mutualized queuing system based on [Apache Kafka](http://kafka.apache.org/).
```

Comment:
I wonder If "shared" would not be a better word for "mutualized" but I ain't native either